decidim-proposals/lib/decidim/proposals/proposal_serializer.rb

```diff
@@ -93,6 +93,16 @@ def original_proposal_url
 
         Decidim::ResourceLocatorPresenter.new(proposal.amendable).url
       end
+
+      def plain_text_body
+        proposal.body.transform_values do |v|
+          if v.is_a? Hash
+            v
+          else
+            Nokogiri::HTML(v).text
```

Comment:
Can we use `decidim_sanitize` here instead? The result would be the same. 

```suggestion
            decidim_sanitize(v, strip_tags: true)
```

You will also need to include these modules:

```diff
diff --git a/decidim-proposals/lib/decidim/proposals/proposal_serializer.rb b/decidim-proposals/lib/decidim/proposals/proposal_serializer.rb
index 7a261b4e1a..cc3a796696 100644
--- a/decidim-proposals/lib/decidim/proposals/proposal_serializer.rb
+++ b/decidim-proposals/lib/decidim/proposals/proposal_serializer.rb
@@ -5,6 +5,8 @@ module Decidim
     # This class serializes a Proposal so can be exported to CSV, JSON or other
     # formats.
     class ProposalSerializer < Decidim::Exporters::Serializer
+      include ActionView::Helpers::SanitizeHelper
+      include Decidim::SanitizeHelper
       include Decidim::ApplicationHelper
       include Decidim::ResourceHelper
       include Decidim::TranslationsHelper
```

Reply:
Thank you for the feedback !